module/submissions/helper.js

```diff
@@ -550,6 +550,877 @@ module.exports = class SubmissionsHelper {
         return answer;
     }
 
+    /**
```

Comment:
@aman-tunerlabs - This has moved to the scoring module, request you to make changes there.